addons/nuxeo-retention/nuxeo-retention-core/src/test/java/org/nuxeo/retention/test/RetentionTestCase.java

```diff
@@ -50,7 +51,7 @@
  * @since 11.1
  */
 @RunWith(FeaturesRunner.class)
-@Features({ TransactionalFeature.class, AutomationFeature.class })
+@Features({ TransactionalFeature.class, AutomationFeature.class, CoreBulkFeature.class })
```

Comment:
`CoreBulkFeature` is declared by `CoreFeature`, you should already have it in the test (no needs to depend on it).